zetaclient/ethrpc/ethrpc.go

```diff
@@ -0,0 +1,137 @@
+package ethrpc
```

Comment:
Adding to the comment below, I think the entire ethRPC can be moved into common

Reply:
Let me make a comment in borrowed files from 3rd-party library.
https://github.com/onrik/ethrpc/blob/main/ethrpc.go

Reply:
the [commit](https://github.com/zeta-chain/node/pull/1665/commits/003e851e10e5f76fee146ac76cc4bb564b6c523b)